COORD/configure/RELEASE

```diff
@@ -0,0 +1,55 @@
+# RELEASE - Location of external support modules
```

Comment:
Can you avoid the use of MASTER_RELEASE as per https://github.com/ISISComputingGroup/IBEX/issues/2320. See https://github.com/ISISComputingGroup/EPICS-ioc/blob/master/ISISDAE/configure/RELEASE for an example. Add this IOC to the ticket when done.

Reply:
done.